File: Leetcode 0401 Binary Watch.go
// 2025/12/15
func readBinaryWatch(turnedOn int) []string {
    ans := []string{}
    for h:=0; h < 12; h++ {
        for m:=0; m <60; m++{
        if bits.OnesCount(uint(h))+bits.OnesCount(uint(m)) == turnedOn{ 
                time := fmt.Sprintf("%d:%02d", h, m)
                ans = append(ans, time)
            }
        }
    }
    return ans
}

File: Leetcode 0401 Binary Watch.py
# 2025/12/15
class Solution:
    def readBinaryWatch(self, turnedOn: int) -> List[str]:
        ans = []
        for i in range(12):
            for j in range(60):
                if i.bit_count() + j.bit_count() == turnedOn:
                    ans.append(f"{i}:{j:02d}")
        return ans
